<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（F)</string>
    </property>
    <addaction name="actionxianjian"/>
    <addaction name="actioniuhh"/>
    <addaction name="separator"/>
    <addaction name="actionAD"/>
    <addaction name="actionAD_2"/>
    <addaction name="separator"/>
    <addaction name="actionADA"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑（E）</string>
    </property>
    <addaction name="actionhjjj"/>
    <addaction name="actionde"/>
    <addaction name="separator"/>
    <addaction name="actiongj"/>
    <addaction name="actiongj_2"/>
    <addaction name="actiongy"/>
    <addaction name="separator"/>
    <addaction name="actionhui"/>
    <addaction name="actionjui"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>运行（R）</string>
    </property>
    <addaction name="actionss"/>
    <addaction name="separator"/>
    <addaction name="actionjjj"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_R"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionShear"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRenew"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionxianjian">
   <property name="icon">
    <iconset>
     <normaloff>../picture/微信图片_20220827173454.jpg</normaloff>../picture/微信图片_20220827173454.jpg</iconset>
   </property>
   <property name="text">
    <string>新建文本 </string>
   </property>
   <property name="toolTip">
    <string>新建文本 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actioniuhh">
   <property name="icon">
    <iconset>
     <normaloff>../picture/微信图片_20220827173536.jpg</normaloff>../picture/微信图片_20220827173536.jpg</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsde">
   <property name="icon">
    <iconset>
     <normaloff>../picture/保存.jpg</normaloff>../picture/保存.jpg</iconset>
   </property>
   <property name="text">
    <string>保存 CTRL+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionderf">
   <property name="text">
    <string>derf</string>
   </property>
  </action>
  <action name="actionAD">
   <property name="icon">
    <iconset>
     <normaloff>../picture/保存.jpg</normaloff>../picture/保存.jpg</iconset>
   </property>
   <property name="text">
    <string>保存 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAD_2">
   <property name="text">
    <string>另存为 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionADA">
   <property name="text">
    <string>关闭所有 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionhjjj">
   <property name="icon">
    <iconset>
     <normaloff>../picture/撤销.jpg</normaloff>../picture/撤销.jpg</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionde">
   <property name="icon">
    <iconset>
     <normaloff>../picture/恢复.jpg</normaloff>../picture/恢复.jpg</iconset>
   </property>
   <property name="text">
    <string>恢复 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actiongj">
   <property name="icon">
    <iconset>
     <normaloff>../picture/剪切.jpg</normaloff>../picture/剪切.jpg</iconset>
   </property>
   <property name="text">
    <string>剪切 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actiongj_2">
   <property name="icon">
    <iconset>
     <normaloff>../picture/复制.jpg</normaloff>../picture/复制.jpg</iconset>
   </property>
   <property name="text">
    <string>复制 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actiongy">
   <property name="icon">
    <iconset>
     <normaloff>../picture/粘贴.jpg</normaloff>../picture/粘贴.jpg</iconset>
   </property>
   <property name="text">
    <string>粘贴 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionhui">
   <property name="icon">
    <iconset>
     <normaloff>../picture/查找.jpg</normaloff>../picture/查找.jpg</iconset>
   </property>
   <property name="text">
    <string>查找 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionjui">
   <property name="icon">
    <iconset>
     <normaloff>../picture/替换.jpg</normaloff>../picture/替换.jpg</iconset>
   </property>
   <property name="text">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionfgh">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../picture/微信图片_20220827173454.jpg</normaloff>../picture/微信图片_20220827173454.jpg</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../picture/微信图片_20220827173536.jpg</normaloff>../picture/微信图片_20220827173536.jpg</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../picture/保存.jpg</normaloff>../picture/保存.jpg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>../picture/复制.jpg</normaloff>../picture/复制.jpg</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>../picture/粘贴.jpg</normaloff>../picture/粘贴.jpg</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionShear">
   <property name="icon">
    <iconset>
     <normaloff>../picture/剪切.jpg</normaloff>../picture/剪切.jpg</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset>
     <normaloff>../picture/查找.jpg</normaloff>../picture/查找.jpg</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset>
     <normaloff>../picture/替换.jpg</normaloff>../picture/替换.jpg</iconset>
   </property>
   <property name="text">
    <string>替换</string>
   </property>
   <property name="toolTip">
    <string>替换</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>../picture/撤销.jpg</normaloff>../picture/撤销.jpg</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionRenew">
   <property name="icon">
    <iconset>
     <normaloff>../picture/恢复.jpg</normaloff>../picture/恢复.jpg</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>../picture/运行.jpg</normaloff>../picture/运行.jpg</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="toolTip">
    <string>运行</string>
   </property>
  </action>
  <action name="actionjjj">
   <property name="icon">
    <iconset>
     <normaloff>../picture/运行.jpg</normaloff>../picture/运行.jpg</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionsw">
   <property name="text">
    <string>sw</string>
   </property>
  </action>
  <action name="actionss">
   <property name="icon">
    <iconset>
     <normaloff>../../ide2.3.3/picture/编译.jpg</normaloff>../../ide2.3.3/picture/编译.jpg</iconset>
   </property>
   <property name="text">
    <string>编译</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
